packages/core/src/html/CustomListIconProcessor.ts

```diff
@@ -100,6 +106,8 @@ IconAttributes | null => {
     size: node.attribs['i-size'] !== undefined ? node.attribs['i-size'] : iconAttrsSoFar?.size,
     className: node.attribs['i-class'] !== undefined ? node.attribs['i-class'] : iconAttrsSoFar?.className,
     spacing: node.attribs['i-spacing'] !== undefined ? node.attribs['i-spacing'] : iconAttrsSoFar?.spacing,
+    oneOff: (node.attribs['i-one-off'] === true || node.attribs['i-one-off'] === 'true')
+      ? true : undefined,
```

Comment:
```suggestion
    oneOff: (node.attribs['i-one-off'] === true || node.attribs['i-one-off'] === 'true')
```

Reply:
Hi, for this it will return True or False instead of True or undefined, which results in the variable being either true, undefined or false, which is not ideal I think?